apps/e2e-test/src/steps/app-specific/workflow/workflow.utils.ts

```diff
@@ -155,3 +155,21 @@ export const addRemoteFile = async (scWorld: ScenarioWorld, filePath: string) =>
   logger.info(`Add remote file source`);
```

Comment:
Please solve the lint issues above.

Reply:
removed the empty methods to fix lint issues.